WordPress/Classes/ViewRelated/Pages/PageListTableViewCell.m

```diff
@@ -145,27 +144,22 @@ - (void)configureBadges
 {
     Page *page = (Page *)self.post;
 
-    NSString *badgesString = @"";
+    NSMutableArray<NSString *> *badges = [NSMutableArray new];
+
+    NSString *timestamp = [self.post isScheduled] ? [self.dateFormatter stringFromDate:self.post.dateCreated] : [self.post.dateCreated mediumString];
+    [badges addObject:timestamp];
     
     if (page.hasPrivateState) {
-        badgesString = NSLocalizedString(@"Private", @"Title of the Private Badge");
+        [badges addObject:NSLocalizedString(@"Private", @"Title of the Private Badge")];
     } else if (page.hasPendingReviewState) {
-        badgesString = NSLocalizedString(@"Pending review", @"Title of the Pending Review Badge");
+        [badges addObject:NSLocalizedString(@"Pending review", @"Title of the Pending Review Badge")];
     }
     
     if (page.hasLocalChanges) {
-        if (badgesString.length > 0) {
-            badgesString = [badgesString stringByAppendingString:@" · "];
-        }
-        badgesString = [badgesString stringByAppendingString:NSLocalizedString(@"Local changes", @"Title of the Local Changes Badge")];
+        [badges addObject:NSLocalizedString(@"Local changes", @"Title of the Local Changes Badge")];
     }
     
-    self.badgesLabel.text = badgesString;
-}
-
-- (void)configureTimeStamp
```

Comment:
This needs to be removed also in `- (void)setPost:(AbstractPost *)post` where `[self configureTimeStamp]`